cfme/v2v/__init__.py

```diff
@@ -0,0 +1,45 @@
+from navmazing import NavigateToSibling
+from widgetastic.widget import Text
+
+from cfme.base import Server
+from cfme.base.login import BaseLoggedInPage
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
+
+
+class MigrationView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
+        return self.logged_in_as_current_user and (
+            self.navigation.currently_selected == ["Compute", "Migration"]
```

Comment:
Q: why two paths ? are those different in 5.9 and 5.10 ?

Reply:
Yes different in 5.9 and 5.10

Reply:
so, I'd rather you used if version here.